catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/SegmentHandler.java

```diff
@@ -104,6 +111,25 @@ private void handleTreValues(Metacard metacard, NitfAttribute attribute, Tre tre
   private Attribute populateAttribute(Metacard metacard, String attributeName, Serializable value) {
     Attribute currentAttribute = metacard.getAttribute(attributeName);
 
+    // check if country code
+    if (COUNTRY_CODE_ATTRIBUTES.contains(attributeName)) {
+      List<Serializable> alpha3CountryCodes = new ArrayList<>();
+      String[] tokenizedCountryCodes = ((String) value).split(" ");
+
+      for (String countryCode : tokenizedCountryCodes) {
+        alpha3CountryCodes.add(CountryCode.getByCode(countryCode).getAlpha3());
```

Comment:
Could this be done using the java Local object instead of this third party one?

Reply:
I'm not sure. Java Locale constructor requires a language code as well, which isn't always easy to get. I read the online api and it looks like the behavior for handing it a junk language code (so we could just use the iso country code functionality) is undefined

Reply:
I'll see if we get around it using its builder pattern
`String cc = (new Locale.Builder()).setRegion(countryCode).build().getISO3Country();`

Reply:
try `new Locale("", countryCode).getISO3Country()` and protect yourself against `MissingResourceException`

Reply:
Yeah it works either passing in an empty string or using the builder